tests/auto/rt_auto.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash --login
+set -eux
+if [ -f "accesstoken.sh" ]; then
+  source ./accesstoken.sh
+  export ghapitoken=$ghapitoken
```

Comment:
Assigning variable to itself is not necessary. In fact exporting `ghapitoken` is not necessary if it's already exported in `accesstoken.sh`.

Reply:
OK. I'm running RT's on all machines right now so i can address this after they're done. Thanks!